.github/CODEOWNERS

```diff
@@ -32,3 +32,12 @@
 /clang/www/make_cxx_dr_status @Endilll
 
 /lldb/ @JDevlieghere
+
+# SPIR-V in MLIR.
+/mlir/include/mlir/Dialect/SPIRV/ @antiagainst @kuhar
+/mlir/lib/Dialect/SPIRV/ @antiagainst @kuhar
+/mlir/include/mlir/Conversion/SPIRVTo*/ @antiagainst @kuhar
+/mlir/include/mlir/Conversion/*ToSPIRV/ @antiagainst @kuhar
+/mlir/lib/Conversion/SPIRVTo*/ @antiagainst @kuhar
+/mlir/lib/Conversion/*ToSPIRV/ @antiagainst @kuhar
+/mlir/tools/mlir-tblgen/SPIRVUtilsGen.cpp @antiagainst @kuhar
```

Comment:
We should also include paths in `/mlir/test` and `/mlir/docs` and `/mlir/unittests`? There are also `/mlir/include/mlir/Target/SPIRV` and `/mlir/lib/Target/SPIRV`. There might also be something else missed; would be good to do `find . -name "*SPIRV*" ` to check out.

Reply:
Done